scripts/imgtool/main.py

```diff
@@ -32,9 +34,9 @@ def gen_ecdsa_p256(keyfile, passwd):
 def gen_ecdsa_p224(keyfile, passwd):
     print("TODO: p-224 not yet implemented")
 
-
-valid_langs = ['c', 'rust']
-keygens = {
+dependencies = None
```

Comment:
I don't think this global should be here, what purpose does it serve?

Reply:
I will remove it.